web/themes/custom/atrium/includes/page.inc

```diff
@@ -0,0 +1,19 @@
+<?php
+
+function atrium_preprocess_page(&$variables) {
+
+  $current_path = \Drupal::service('path.current')->getPath();
+  $path_segments = explode('/', $current_path);
+  $eid = $path_segments[2];
+
+  if ($path_segments[1] == 'media' || $path_segments[1] == 'node') {
+    $entity_type = $path_segments[1];
+
+    /** @var $entity \Drupal\media\Entity\Media */
+    /** @var $entity \Drupal\node\Entity\Node */
+    $entity = \Drupal::entityTypeManager()->getStorage($entity_type)->load($eid);
+    $entity_bundle = $entity->bundle();
+    $variables['entity_bundle'] = $entity_bundle;
+  }
+
+}
```

Comment:
I needed to add some PHP so that we can get the media type ('jcc_forms') in the page.tpl.php file. I included the node logic so there could be consistency in page.tpl.php.